src/TTGamesExplorerRebirthUI/Forms/FontForm.cs

```diff
@@ -98,7 +99,11 @@ private void DarkComboBox1_SelectedValueChanged(object sender, EventArgs e)
         {
             if (!darkCheckBox1.Checked)
             {
-                int indexChar = int.Parse(darkComboBox1.SelectedItem.ToString().Replace("Char #", "")) - 1;
+                int indexChar = 0;
+                if (darkComboBox1.SelectedItem.ToString() != "None")
+                {
+                    indexChar = int.Parse(darkComboBox1.SelectedItem.ToString().Replace("Char #", "")) - 1;
```

Comment:
I'm not sure if the `- 1` is still needed here since there is `None` at index 0.
Additionally, if you could do the same changes to TSHForm, that could be nice!